ui/views_test.py

```diff
@@ -208,7 +208,7 @@ def test_roles_setting(self):
             {
                 'program': role.program.id,
                 'role': role.role,
-                'permissions': [key for key, value in available_perm_status(profile.user).items()],
+                'permissions': [key for key, value in available_perm_status(profile.user).items() if value is True],
```

Comment:
It seems reasonable to me (even if it does not make much sense if the  a value = False is not expected in the rest of the code).